web/styles/subscriptions.css

```diff
@@ -401,6 +401,10 @@ h4.user_group_setting_subsection_title {
                 display: block;
                 margin: 0 auto 10px;
             }
+
+            .create-stream-disabled-text {
```

Comment:
Can this be removed and use `settings-empty-option-text` instead in the html element?